lighthouse-cli/test/smokehouse/test-definitions/redirects/expectations.js

```diff
@@ -27,6 +27,9 @@ const expectations = [
           },
         },
       },
+      runWarnings: [
+        /The page may not be loading as expected because your test URL \(.*online-only.html.*\) was redirected to .*redirects-final.html. Try testing the second URL directly./,
```

Comment:
is there a non-deterministic component of this that warrants a regex?

Reply:
this first one - yes, `cacheBuster` is a new Date. The second - no. Still good for eliding the URL query params that don't matter.